src/config/resolve-config-editorconfig.js

```diff
@@ -3,11 +3,9 @@
 const path = require("path");
 
 const editorconfig = require("editorconfig");
-const mem = require("mem");
 const editorConfigToPrettier = require("editorconfig-to-prettier");
 const findProjectRoot = require("./find-project-root.js");
-
```

Comment:
nit: why remove the blank line here?

I would generally keep a blank line between a require and any other code.

Reply:
think it was just lost during a few different attempts at solving the problem, happy to put it back